### PR TITLE
Shortest common supersequence

### PR DESCRIPTION
**Pull request by bitaashna**
Implemented the code of Shortest common supersequence  in C++
**#167** 
[x] - Code is well indented.
[x] - Code is well commented.
[x] - Code is not plagiarized.
[x] - Code is placed in the proper folder.
[x] - The readme.md of the algorithm folder is updated with the code link.
**#GSSOC'21**

### CODE REVIEW

**Comment by bitaashna** [comment]
As i have made another folder. That's why i have commited the file again and created the PR again. Please review it
 #167 #gssoc'21
@rudrakshi99 @plazzy99

File: Code/C++/print_shortest_supersequence.cpp [hunk 1]
```diff
@@ -0,0 +1,173 @@
+/*
+* Shortest Common Supersequence finds the shortest supersequence which consist of all the characters of both the string in the order specified.
```
Comment (plazzy99) [inline Code/C++/print_shortest_supersequence.cpp:2]: This program is not linked with the issue #167 
so raise it in another pr

**Review by plazzy99 — CHANGES_REQUESTED** [review]
1 pr resolves 1 issue

File: Code/C++/print_shortest_supersequence.cpp (hunk 1 above)
Comment (bitaashna) [reply Code/C++/print_shortest_supersequence.cpp:2]: @plazzy one file for printing supersequence and  and one file for printing length has been attached.It is related to issue only.
please check it one more time

**Pull request closed without merging**